docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -322,6 +322,132 @@ their values independently at runtime.
         self.value = new_value
     ```
 
+#### :octicons-code-16: Separating parameters during compilation
+
+In practical scenarios, it is usually unfeasible to embed model parameters
+in the IR.
+
+`aot.externalize_module_parameters()` allows for the separation of
+the parameters from the IR, but still encodes a symbolic relationship between
+between parameters and the IR, so they can be loaded at runtime.
```

Comment:
This first sentence is a bit awkward ("unfeasible") and focuses on internal implementation details ("IR" intermediate representation is mostly internal to the compiler).

Borrowing some language from https://iree.dev/guides/parameters/, I'd rephrase to focus on what benefits external parameters provide and maybe use terms like "program"/"code"/"compute graph" instead of "IR". Acronyms can also be overwhelming for users, especially when the acronyms come from different domains.

This should also link to `../parameters.md` for more information somewhere near the top.

```suggestion
Model parameters can be stored in standalone files that can be efficiently
stored and loaded separately from model compute graphs. See the
[Parameters guide](../parameters.md) for more general information about
parameters in IREE.

When using iree-turbine, the `aot.externalize_module_parameters()` function
separates parameters from program modules and encodes a symbolic relationship
between them so they can be loaded at runtime.
```